pyalgotrading/algobulls/api.py

```diff
@@ -278,19 +285,17 @@ def stop_strategy_algotrading(self, strategy_code: str, trading_type: TradingTyp
             `POST` v1/customer_strategy_algotrading
         """
         if trading_type == TradingType.REALTRADING:
-            endpoint = 'v2/portfolio/strategies'
-        elif trading_type == TradingType.PAPERTRADING:
-            endpoint = 'v2/papertrading/strategies'
-        elif trading_type == TradingType.BACKTESTING:
-            endpoint = 'v2/backtesting/strategies'
+            return {'message': 'Please get approval for your strategy by writing to support@algobulls.com. Once approved, you can STOP the strategy in REALTRADING mode directly from the website.'}
+        elif trading_type in [TradingType.PAPERTRADING, TradingType.BACKTESTING]:
+            endpoint = 'v4/portfolio/strategies'
         else:
             raise NotImplementedError
 
         try:
             key = self.__get_key(strategy_code=strategy_code, trading_type=trading_type)
-            json_data = {'method': 'update', 'newVal': 0, 'key': key, 'record': {'status': 2}}
+            json_data = {'method': 'update', 'newVal': 0, 'key': key, 'record': {'status': 2}, 'dataIndex': 'executeConfig'}
```

Comment:
@pushpakdagade 

the keys newVal, key, record, dataIndex must be required by the other end
but they dont seem standardized

please see if we want to change this at the other end, and then here, else kindly ignore